src/core/client/stream/tabs/Profile/CommentHistory/HistoryComment.css

```diff
@@ -1,13 +1,33 @@
+.root:not(:first-child) {
+  border-top: 1px solid var(--palette-divider);
+  padding-top: var(--spacing-3)
+}
+
 .icon {
   color: var(--palette-text-secondary);
   margin-right: calc(0.5 * var(--mini-unit));
 }
 
 .replies {
   composes: button from "coral-ui/shared/typography.css";
+  display: inline-flex;
+  margin-right: var(--spacing-3)!important;
```

Comment:
You seem to use `!important` here to overwrite the gutter that is provided the `Flex` layouting component. Instead you should just remove the gutter from the `Flex` Component.

Reply:
Oh I didn't realize flex had a configurable gutter.